package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "angular-feincms",
+  "dependencies": {},
+  "devDependencies": {
+    "grunt": "0.4.5",
```

Comment:
May as well use the latest of everything here =)

Specifically this must be v1 because v1 is being used for concat and uglify, whereas watch needs to be v1 too